content/cfp.md

```diff
@@ -44,7 +44,8 @@ We are looking for:
 
 **We'll announce a new deadline soon**
 
-Submit before **March 1st 2022 23:59:59 UTC+2**. The final program will be announced on April 1st (perhaps with a joke).
+Submit before **September 10th 2023 23:59:59 UTC+2**.
+The final program will be announced on September 1st.
```

Comment:
This is an error right? We want the submissions before we announce the final program right?

Reply:
Oh yes definitely, is August 10th good?

Reply:
I picked August 21st so a bit more time to emerge from holidays and a bit of buffer to postpone the deadline if necessary :+1: